### ISSUE TITLE
Compatibility with R

### ISSUE THREAD

**Issue by sohitmiglani** [jipingw/RiboDiPA#3]
Hello,

Thank you for making this tool. I was trying to install and it seems like you've listed that it requires R version 4.1 at minimum according to your DESCRIPTION file (Depends: R (>= 4.1) ). 

But the most latest version of R released is R 4.0.3 so I believe it's a typo in your DESCRIPTION file ? Did you mean 4.0.1 ?

As of now, it won't let me install your package because I have R 4.0.3 and your package requires R > 4.1

**Comment by jipingw** [comment]
@sohitmiglani you are right. when I submitted to bioconductor for review, it required me to specify R>=4.1, otherwisei I got warnings. Here is the solution. download the package from github.  Open DESCRIPTION file, change R>=4.1 to R>=4.0. use R CMD build and RCMD install from command line. You should be able to install it without a problem. If you encounter problem, please let me know.
